jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,6 +1,9 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
```

Comment:
Should we have a sprint next week and work out the details for all schemas at once?